components/Timezone Selector/index.tsx

```diff
@@ -24,9 +27,18 @@ const TimezoneSelector: React.FC<TimezoneSelectorProps> = ({ value }) => {
         })
         if (res.status === 200) {
           setSelectedTimezone(selectedOption.value)
+          setError('')
+          setSuccess('Timezone updated successfully.')
         }
       } catch (err: any) {
```

Comment:
this should actually be at the else of the `if ` above. If the api runs into an error, it will return a status code 500 response message, not throw here.